### PR TITLE
[FIX] acc_pay_*_patch: correct payment status on invoices portal

### PR DESCRIPTION
**Pull request by Feyensv**
---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account_payment_invoice_online_payment_patch/views/account_portal_templates.xml [hunk 1]
```diff
@@ -23,6 +23,12 @@
                     <t t-if="invoice.state == 'posted' and invoice.payment_state in ('paid', 'in_payment') or last_tx.state == 'done'">
                         <span class="badge rounded-pill text-bg-success"><i class="fa fa-fw fa-check"></i><span class="d-none d-md-inline"> Paid</span></span>
                     </t>
+                    <t t-if="invoice.state == 'posted' and invoice.payment_state == 'reversed'">
+                        <span class="badge rounded-pill text-bg-success"><i class="fa fa-fw fa-check"></i><span class="d-none d-md-inline"> Reversed</span></span>
+                    </t>
+                    <t t-if="invoice.state == 'cancel'">
+                        <span class="badge rounded-pill text-bg-danger"><i class="fa fa-fw fa-remove"></i><span class="d-none d-md-inline"> Cancelled</span></span>
```
Comment (AntoineVDV) [inline addons/account_payment_invoice_online_payment_patch/views/account_portal_templates.xml:30]:
```suggestion
                        <span class="badge rounded-pill text-bg-danger"><i class="fa fa-fw fa-remove"></i><span class="d-none d-md-inline"> Canceled</span></span>
```

**Review by AntoineVDV — COMMENTED** [review]
@robodoo delegate+

File: addons/account_payment_invoice_online_payment_patch/views/account_portal_templates.xml (hunk 1 above)
Comment (Feyensv) [reply addons/account_payment_invoice_online_payment_patch/views/account_portal_templates.xml:30]: Same problem some lines below, and wanna keep c/p with existing xml code.

**Comment by Feyensv** [comment]
@robodoo r+

**Pull request closed without merging**